com.unity.cinemachine/Editor/Editors/CinemachineConfiner2DEditor.cs

```diff
@@ -48,24 +51,27 @@ void TrackVolume(SerializedProperty p)
                 polygonsHelp.SetVisible(cc != null && cc.geometryType != CompositeCollider2D.GeometryType.Polygons);
             }
 
-            var confiner = Target; // so it gets captured in the lambdas
             var bakeProgress = ux.AddChild(new ProgressBar { lowValue = 0, highValue = 100 });
             var bakeTimeout = ux.AddChild(new HelpBox(
                 "Polygon skeleton computation timed out.  Confiner result might be incomplete."
                 + "\n\nTo fix this, reduce the number of points in the confining shape, "
                 + "or set the MaxWindowSize parameter to limit skeleton computation.",
                 HelpBoxMessageType.Warning));
-
+            
+            var confiner = Target; // so it gets captured in the lambdas
             UpdateBakingProgress();
             ux.schedule.Execute(UpdateBakingProgress).Every(250); // GML todo: is there a better way to do this?
             void UpdateBakingProgress()
             {
+                oversizedCameraHelp.SetVisible(false);
                 if (confiner == null)
                     return; // target deleted
+                
                 if (!confiner.OversizeWindow.Enabled)
                 {
                     bakeTimeout.SetVisible(false);
                     bakeProgress.SetVisible(false);
+                    oversizedCameraHelp.SetVisible(confiner.IsCameraOversizedForTheConfiner());
```

Comment:
This is not appearing and disappearing for me when I change the lens size.  I have to slide something in the UX to trigger it (e.g. change the game window size).  Unity 2022.2f1.

Reply:
I can't reproduce the issue. I have to click Invalidate Lens Cache to make the text box appear/disappear.
Which makes me think:
We should check whether the Lens Cache is valid in the editor script. Let them know, with a warning, that the lens cache is not valid, they should invalidate it.